lib/api/client-commands/pause.js

```diff
@@ -25,26 +27,48 @@ util.inherits(Pause, EventEmitter);
 
 Pause.prototype.command = function(ms, cb) {
   // If we don't pass the milliseconds, the client will
-  // be suspended indefinitely
+  // be suspended indefinitely, until the user presses some
+  // key in the terminal to resume it.
   if (!ms) {
-    ms = 1000 * 60 * 60 * 6; // 6 hours
-    // eslint-disable-next-line
-    console.log('Waiting... Press Ctrl+C to exit.');
-    process.on('SIGINT', (code) => {
-      this.api.end(function () {
-        process.exit(0);
-      });
-    });
-  }
+    /* eslint-disable */
+console.log(`
+  Paused...
+  Press <space> to step over to the next test step and pause again.
```

Comment:
The common shortcut key for step over is `F10` so I think we should also add that.